src/closedwindows/closedcard.less

```diff
@@ -0,0 +1,22 @@
+.closedtime {
+  font-size: 12px;
+  font-weight: 100;
```

Comment:
We have variables for font weights (this might be `@font-light`? not sure which though.)